src/util.rs

```diff
@@ -98,7 +98,7 @@ pub fn diff_paths<P, B>(path: P, base: B) -> Option<PathBuf>
         }
     }
 
-    Some(comps.iter().map(|c| c.as_os_str()).collect::<PathBuf>().into())
+    Some(comps.iter().map(|c| c.as_os_str()).collect())
```

Comment:
It's up for debate if removing the turbofish here is a good idea. It might make it less clear, you'll have to look at the function signature to know the type it's collecting into.

Reply:
I'm fine with it.